index.bs

```diff
@@ -643,6 +643,21 @@ partial interface Gamepad {
 <dfn attribute for="Gamepad" id="gamepad-getvrdisplays-attribute">displayId</dfn>
 Return the {{VRDisplay/displayId}} for the associated {{VRDisplay}}.
 
+# Processing Model # {#proc}
+
+## submitFrame ## {#submitframe-proc}
+
+The are are multiple situations where applications won't invoke submitFrame at the expected rate (the VRDisplay native framerate). The User Agent must intervene to protect the user from motion sickness or to consistently inform her about different situations like errors or navigation between pages. This is the breakdown of the different scenarios:
```

Comment:
* typo: `are are`
* can you change `VRDisplay` to `{{VDisplay}}` and `submitFrame` to `{{VRDisplay}}.{{submitFrame()}}` so Bikeshed can properly link things up?
* `The User Agent must` to `The User Agent MUST`
* `or to consistently` to `or consistently`
* `different situations like` to `different situations such as`
* personally, I would change `about different situations like errors or navigation between pages. This is the breakdown of the different scenarios:` to `about such scenarious:`